instructure/canvas-ios
# Preserve scroll position in Modules

refs: MBL-14346
affects: student, teacher
release note: none

Test plan:
- Install Student app and enable `student_modules` feature
- narmstrong > CS 1400 > Modules > Sequential > None submission types
- Tap kebab in nav bar and toggle "mark as done" requirement"
- Go back to the list
- The cell should update and scroll position should be preserved
- Go to Pages > Links to Stuff
- Tap link to Module 1
- Should auto-scroll to Module 1
- Collapse and expand sections to make sure that all works
- View modules in Teacher app as well to make sure it's all good there
